tpot/tpot.py

```diff
@@ -288,6 +289,182 @@ def score(self, training_features, training_classes, testing_features, testing_c
 
         return self._evaluate_individual(self.optimized_pipeline_, training_testing_data)[0]
 
+    def export(self, output_file_name):
+        """Exports the current optimized pipeline as Python code.
+
+        Parameters
+        ----------
+        output_file_name: string
+            String containing the path and file name of the desired output file
+
+        Returns
+        -------
+        None
+        """
+        if self.optimized_pipeline_ == None:
+            raise Exception('A pipeline has not yet been optimized. Please call fit() first.')
+        
+        exported_pipeline = self.optimized_pipeline_
+        
+        # Replace all of the mathematical operators with their results
+        while True:
```

Comment:
Looking at the while loop again, I think I have it there because the code is constantly changing the contents of the list it's iterating over, so it's safer to restart the iterator over again when the list is changed. That probably doesn't matter since we're iterating over the list backwards, but I'll leave that for a future revision of the export method (since it's pretty fast right now anyway).